utils/capellacontrol/controller.go

```diff
@@ -1107,3 +1169,25 @@ func (c *Controller) GetTrustedCAs(
 
 	return resp, err
 }
+
+type CreateUpdateServerVersionRequest struct {
```

Comment:
I think this should be just `UpdateServerVersionRequest`?

Reply:
yes, updated